src/AdePTGeant4Integration.cpp

```diff
@@ -202,32 +202,17 @@ void AdePTGeant4Integration::InitScoringData(adeptint::VolAuxData *volAuxData)
       G4TransportationManager::GetTransportationManager()->GetNavigatorForTracking()->GetWorldVolume();
   const vecgeom::VPlacedVolume *vecgeomWorld = vecgeom::GeoManager::Instance().GetWorld();
 
-  // Used to keep track of the current vecgeom history while visiting the tree
-  std::vector<vecgeom::VPlacedVolume const *> aCurrentVecgeomHistory;
-  // Used to keep track of the current geant4 history while visiting the tree
-  std::vector<G4VPhysicalVolume const *> aCurrentGeant4History;
-
   // recursive geometry visitor lambda matching one by one Geant4 and VecGeom logical volumes
   typedef std::function<void(G4VPhysicalVolume const *, vecgeom::VPlacedVolume const *)> func_t;
   func_t visitGeometry = [&](G4VPhysicalVolume const *g4_pvol, vecgeom::VPlacedVolume const *vg_pvol) {
     const auto g4_lvol = g4_pvol->GetLogicalVolume();
     const auto vg_lvol = vg_pvol->GetLogicalVolume();
 
-    aCurrentVecgeomHistory.push_back(vg_pvol);
-    aCurrentGeant4History.push_back(g4_pvol);
-
-    // If the volume is sensitive:
-    if (volAuxData[vg_lvol->id()].fSensIndex == 1) {
-      // Initialize mapping of Vecgeom sensitive PlacedVolume IDs to G4 PhysicalVolume IDs
-      // In order to be able to reconstruct navigation histories based on a VecGeom Navigation State Index,
-      // we need to map not only the sensitive volume, but also the ones leading up to here
-      for (uint i = 0; i < aCurrentVecgeomHistory.size() - 1; i++) {
-        fglobal_vecgeom_to_g4_map.insert(
-            std::pair<int, const G4VPhysicalVolume *>(aCurrentVecgeomHistory[i]->id(), aCurrentGeant4History[i]));
-      }
-      bool new_pvol =
-          fglobal_vecgeom_to_g4_map.insert(std::pair<int, const G4VPhysicalVolume *>(vg_pvol->id(), g4_pvol)).second;
-    }
+    // Initialize mapping of Vecgeom sensitive PlacedVolume IDs to G4 PhysicalVolume IDs
```

Comment:
```suggestion
    // Initialize mapping of Vecgeom PlacedVolume IDs to G4 PhysicalVolume IDs
```

Although the second sentence is already clear, let's cut the sensitive here to avoid confusion :)